### PR TITLE
fix(set): remove deprecated set component

### PR DESCRIPTION
**Pull request by lucasaarcoverde**
#### What is the purpose of this pull request?

As the title says.

#### What problem is this solving?

Set is a deprecated component, you should use `Stack` instead.

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

**Review by marcelovicentegc — COMMENTED** [review]
LGTM but there is a line that needs to be removed at `packages/admin-ui/src/index.ts` 😊

File: packages/admin-ui-docs/docs/guidelines/components/dropdown.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ function Example() {
   })
 
   return (
-    <Set spacing={3}>
+    <Stack space="$l">
```
Comment (lucasaarcoverde) [inline packages/admin-ui-docs/docs/guidelines/components/dropdown.mdx:64]:
```suggestion
    <Stack space="$l" direction="row">
```

File: packages/admin-ui-docs/docs/guidelines/components/dropdown.mdx [hunk 2]
```diff
@@ -97,14 +97,14 @@ function Example() {
...
```
Comment (lucasaarcoverde) [inline packages/admin-ui-docs/docs/guidelines/components/dropdown.mdx:100]:
```suggestion
    <Stack space="$l" direction="row">
```

File: packages/admin-ui-docs/docs/guidelines/experimental/combobox.mdx [hunk 3]
```diff
@@ -344,14 +338,14 @@ function Example() {
 
   return (
     <>
-      <Set>
+      <Stack direction="row">
```
Comment (lucasaarcoverde) [inline packages/admin-ui-docs/docs/guidelines/experimental/combobox.mdx:341]: this is the only change here, the other ones are just space formatting

File: packages/admin-ui-docs/src/components/IconsGrid/index.tsx [hunk 4]
```diff
@@ -97,7 +97,7 @@ export function IconsGrid(props: IconsGridProps) {
         />
       </DataViewControls>
       {selectedWeight === 'Fill' || selectedSize === 'Small' ? (
-        <Alert visible csx={{ marginBottom: 4 }}>
+        <Alert csx={{ marginBottom: 4 }}>
```
Comment (anitavincent) [inline packages/admin-ui-docs/src/components/IconsGrid/index.tsx:100]: Why?

**Review by anitavincent — APPROVED** [review]

File: packages/admin-ui-docs/src/components/IconsGrid/index.tsx (hunk 4 above)
Comment (lucasaarcoverde) [reply packages/admin-ui-docs/src/components/IconsGrid/index.tsx:100]: this prop doesn't exist anymore, alert is always visible now

**Review by Lenderson — APPROVED** [review]

**Pull request merged**